admin-dev/themes/new-theme/js/nav_bar.js

```diff
@@ -86,12 +88,30 @@ export default class NavBar {
           $submenu.find('i.material-icons.sub-tabs-arrow').text('keyboard_arrow_up');
         });
 
+      function showNavBarContent(event) {
+        if (event.propertyName == 'width') {
+          $navBar[0].removeEventListener(endTransitionEvent, showNavBarContent);
+
+          $('.main-menu').toggleClass('sidebar-closed');
+
+          transitionFired = false;
+        }
+      }
+
       $navBar.on(
         'click',
         '.menu-collapse',
         function onNavBarClick() {
           $('body').toggleClass('page-sidebar-closed');
 
+          if(!transitionFired) {
```

Comment:
Watch out for spaces
```suggestion
          if (!transitionFired) {
```